samples/client/petstore/java/src/main/java/io/swagger/client/ApiClient.java

```diff
@@ -237,6 +238,67 @@ public String parameterToString(Object param) {
     }
   }
 
+  /*
+    Format to {@code Pair} objects.
+  */
+  public List<Pair> parameterToPairs(String collectionFormat, String name, Object value){
+    List<Pair> params = new ArrayList<Pair>();
+
+    // preconditions
+    if (name == null || name.isEmpty()) return params;
+
+    if (value == null) {
+      params.add(new Pair(name, ""));
+      return params;
+    }
```

Comment:
when value is `null`, it should not be included in the `params` (with a value of empty string), it should be ignored completely (the same with when `name` is `null`)